src/v2/view-builder/views/signin/SignInWithDeviceOption.js

```diff
@@ -73,5 +72,15 @@ export default View.extend({
     return {
       signInWithDeviceIsRequired: !!this.options.isRequired,
     };
+  },
+
+  postRender() {
+    if(this.options.isRequired) {
+      const app = this.options.appState.attributes.app;
+      const resourceLabel = app ? loc('oktaVerify.deviceOnlyDescription', 'login', [app.label]) : 
```

Comment:
Added this condition in case there is no app. If we have app always in device only case, can remove.

Reply:
Verified. Looks like some type of apps we do not send app in the response from the server to SIW. So will have to use the default "resource" label.
![Screen Shot 2022-05-11 at 3 34 41 PM](https://user-images.githubusercontent.com/100624019/167959331-ac14ba0e-5df1-41e5-9bde-1fbb35ed766d.png)

@stevennguyen-okta can you please review if this is fine with you!

Reply:
can we have the default be `this resource` rather than `resource`? to improve the grammar. let me know what you think

Reply:
Sure.